rero_ils/modules/documents/jsonschemas/documents/document_contribution_local-v0.0.1.json

```diff
@@ -0,0 +1,59 @@
+{
+  "title": "Local Entity",
```

Comment:
Maybe better to name `Local Contribution`?

Reply:
The idea is to express the fact that a contribution is an entity. The professional should "see" that an entity can be a subfield of different document fields.